rhcos4/cpe/rhcos4-cpe-dictionary.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd">
+      <cpe-item name="cpe:/a:redhat:openshift_container_platform:4.1">
+            <title xml:lang="en-us">Red Hat OpenShift Container Platform 4</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_app_is_ocp4</check>
```

Comment:
```suggestion
      <cpe-item name="cpe:/a:redhat:enterprise_linux_coreos:4.1">
            <title xml:lang="en-us">Red Hat Enterprise Linux CoreOS release 4</title>
            <!-- the check references an OVAL file that contains an inventory definition -->
            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_app_is_rhcos4</check>
```

Reply:
You need to create this file: `installed_app_is_rhcos4.xml` under `shared/checks/oval/`, similar to: `shared/checks/oval/installed_app_is_ocp4.xml`

Reply:
Done